src/main.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { App } from './App';
+import './index.scss';
```

Comment:
Precisa estar importado mesmo não sendo utilzado explicitamente?

Reply:
It's used in front-end build.